dspace/config/dspace.cfg

```diff
@@ -1488,6 +1488,9 @@ mail.helpdesk.name = Help Desk
 # Should all Request Copy emails go to the helpdesk instead of the item submitter?
 request.item.helpdesk.override = false
 
+### metadata-deletion script configuration ###
+bulkedit.allow-bulk-deletion = dspace.agreements.end-user
```

Comment:
One tiny comment: This configuration should be in the `bulkedit.cfg` (since it starts with that prefix) instead of in `dspace.cfg`.

https://github.com/DSpace/DSpace/blob/main/dspace/config/modules/bulkedit.cfg

I'd also recommend a more detailed description in that file.  Something like:
```
### metadata-deletion script configuration ###
# Comma separated list of metadata fields which can be deleted (in bulk) by the 'metadata-deletion' script.
# By default, only 'dspace.agreements.end-user' can be deleted in bulk, as doing so allows
# an administrator to force all users to re-review the End User Agreement on their next login.
```